packages/@pollyjs/adapter-fetch/src/index.js

```diff
@@ -8,10 +8,16 @@ const IS_STUBBED = Symbol();
 const REQUEST_ARGUMENTS = Symbol();
 
 export default class FetchAdapter extends Adapter {
-  static get name() {
+  static get id() {
     return 'fetch';
   }
 
+  static get name() {
+    // NOTE: deprecated in 4.1.0 but proxying since it's possible "core" is behind
+    // and therefore still referencing `name`.  Remove in 5.0.0
+    return this.id;
```

Comment:
Should we just release a new major version instead of having to deal with all of these deprecations?

Reply:
It's a minor change imo and will help anyone with custom adapters to migrate without reading docs.